Assets/!/Scripts/PlayerAudio.cs

```diff
@@ -0,0 +1,68 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using System.IO;
+using utils;
+
+public class PlayerAudio : MonoBehaviour
+{
+    public AudioSource audioSource;
+    public List<Audio> audioPlayer = new List<Audio>();

```

Comment:
We can just use an array because this will be a fixed length.
```suggestion
    public Audio[] audioPlayer;

```